webpack/components/Table/TableWrapper.js

```diff
@@ -27,19 +28,20 @@ const TableWrapper = ({
   const dispatch = useDispatch();
   const { foremanPerPage = 20 } = useForemanSettings();
   // setting pagination to local state so it doesn't disappear when page reloads
-  const [perPage, setPerPage] = useState(foremanPerPage);
-  const [page, setPage] = useState(1);
-  const [total, setTotal] = useState(0);
+  const [perPage, setPerPage] = useState(Number(metadata?.per_page ?? foremanPerPage));
+  const [page, setPage] = useState(Number(metadata?.page ?? 1));
+  const [total, setTotal] = useState(Number(metadata?.subtotal ?? 0));
+
 
   const updatePagination = (data) => {
     const { subtotal: newTotal, page: newPage, per_page: newPerPage } = data;
-    if (newTotal !== undefined) setTotal(parseInt(newTotal, 10));
-    if (newPage !== undefined) setPage(parseInt(newPage, 10));
-    if (newPerPage !== undefined) setPerPage(parseInt(newPerPage, 10));
+    if (newTotal !== undefined) setTotal(Number(newTotal));
+    if (newPage !== undefined) setPage(Number(newPage));
+    if (newPerPage !== undefined) setPerPage(Number(newPerPage));
```

Comment:
I did see my perPage change from `42` to `"42"`. If you can think of an elegant way to validate that, might be nice to do it here